### PR TITLE
exposed more args to user

### PR DESCRIPTION
**Pull request by dcchivian**

### CODE REVIEW

File: lib/hipmer/hipmerUtils.py [hunk 1]
```diff
@@ -70,6 +70,25 @@ def _validate_inputs(self, params):
         # just in case I need the list someday.
         params['mer_sizes_int'] = mer_sizes_int
 
+        # Parse the scaff kmer string into a list.  This step verifies that there
+        # were integers separated by commas. I don't actually use the created list (scaff_mer_sizes_int).
+        if params.get('scaff_mer_lens'):
```
Comment (scanon) [inline lib/hipmer/hipmerUtils.py:75]: Should we have a default if this were empty?

**Review by scanon — COMMENTED** [review]
LGTM.  One minor comment/question.

File: lib/hipmer/hipmerUtils.py (hunk 1 above)
Comment (dcchivian) [reply lib/hipmer/hipmerUtils.py:78]: @scanon there's a default in spec.json if someone invokes from narrative.  you're right though, we should message if it's not set.  I want call to include explicit params, so not setting val with default in code, but am recommended default vals.  OK?

**Comment by dcchivian** [comment]
@scanon cleaned up App Docs a bit.  Added MHM paper to references

**Review by scanon — APPROVED** [review]
LGTM

**Pull request merged**